crates/re_renderer/src/context.rs

```diff
@@ -204,6 +200,17 @@ impl RenderContext {
     fn poll_device(&mut self) {
         crate::profile_function!();
 
+        // Browsers don't let us wait for GPU work via `poll`.
+        // * WebGPU: `poll` is a no-op as the spec doesn't specify it at all.
+        // * WebGL: Internal timeout on can't go above a browser specific value.
+        //          Since wgpu ran into issues in the past with some browsers returning errors,
+        //          it uses a timeout of zero and ignores errors there.
+        //          TODO(andreas): That's not the only thing that's weird with `maintain` in general.
+        //                          See https://github.com/gfx-rs/wgpu/issues/3601
```

Comment:
```suggestion
        // * WebGL: Internal timeout can't go above a browser specific value.
        //          Since wgpu ran into issues in the past with some browsers returning errors,
        //          it uses a timeout of zero and ignores errors there.
        //          TODO(andreas): That's not the only thing that's weird with `maintain` in general.
        //                          See https://github.com/gfx-rs/wgpu/issues/3601
```